datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -31,6 +34,28 @@
 }
 
 
+def create_codeowners_resolver():
+    """Creates an object that resolves owners for files in a repo."""
+    owners_resolver = CodeOwners("\n".join(get_codeowners()))
+    return owners_resolver
+
+
+def validate_logs_assets_codeowners():
+    """Validate `CODEOWNERS` assigns logs as owner for all log assets."""
+
+    has_failed = False
```

Comment:
Let's turn this into a list of failures and return that instead of the boolean. We can then loop over this list and do the echoing in `codeowners` function. This is just to separate IO a little bit from the "business logic".

Reply:
Thanks ! That makes sense to me, I've added that in : 456f7c73b1